src/sentry/eventstream/kafka/backend.py

```diff
@@ -69,9 +71,17 @@ def run_post_process_forwarder(
         synchronize_commit_group,
         commit_batch_size=100,
         initial_offset_reset="latest",
+        event_type=None,
     ):
         logger.debug("Starting post-process forwarder...")
 
+        topic_config = None
+
+        topic_config = settings.SENTRY_EVENTSTREAM_TOPICS.get(
+            event_type, settings.SENTRY_EVENTSTREAM_DEFAULT_TOPIC
+        )
+        topic_name = settings.KAFKA_TOPICS[topic_config]["topic"]
```

Comment:
It might be worth consolidating this into a `__get_topic_for_event_type(event_type: str) -> str` method?